ext/afform/core/ang/af/afForm.component.js

```diff
@@ -197,10 +199,9 @@
           }
         })
         .catch(function(error) {
-          status.resolve();
-          status = CRM.status(error.error_message, 'error');
+          status.reject();
           $element.unblock();
-          CRM.alert(error.error_message, ts('Form Error'));
+          CRM.alert(error.error_message || '', ts('Form Error'));
```

Comment:
changes in this `catch` block seem like they are pretty independent of the larger PR and could be merged quickly as a separate PR